packages/@lwc/shared/src/language.ts

```diff
@@ -41,26 +45,33 @@ const {
     sort: ArraySort,
     splice: ArraySplice,
     unshift: ArrayUnshift,
-    forEach,
 } = Array.prototype;
```

Comment:
Alphabetized.

It bugs me that `forEach` is the one Array function that isn't called `Array*`, but I did not bother to fix that.